src/field/scrcmd_pokemon_misc.c

```diff
@@ -929,7 +929,7 @@ BOOL ScrCmd_CreatePokeathlonFriendshipRoomStatues(ScriptContext *ctx) {
     SavePokeathlonFriendshipRecords *unkPtr = sub_02031B00(fieldSystem->saveData);
 
     for (i = 0; i < 3; i++) {
-        LocalMapObject *mapObj = GetMapObjectByID(fieldSystem->mapObjectMan, 0xf6 + i);
+        LocalMapObject *mapObj = GetMapObjectByID(fieldSystem->mapObjectManager, 0xf6 + i);
```

Comment:
`#define POKEATHLON_FRIENDSHIP_STATUE_MAPOBJECT_ID       246`

Reply:
which file should this be in

Reply:
not sure yet \:)

Reply:
I'll probably just leave this then, unless anyone else has a suggestion